src/app.js

```diff
@@ -1 +1,24 @@
 'use strict';
+
+const terminal = require('./modules/terminal');
+const fs = require('fs');
+
+function copyFile() {
+  terminal.question('Enter your command: ', (key) => {
+    const [command, firstFile, secondFile] = key.split(' ');
+
+    if (command === 'cp' && !fs.existsSync(secondFile)) {
+      fs.copyFile(firstFile, secondFile, (err) => {
+        if (err) {
+          throw new Error('Error!');
```

Comment:
It's better to print the actual error message instead of a generic 'Error!'. This would help you debug if anything goes wrong.